<launch>
    <!-- promot -->
    <arg name="tools_config" default="$(find tello_llm_ros)/config/tools.json"/>
    <arg name="common_prompt" default="$(find tello_llm_ros)/config/prompts/common_system_prompt-EN.txt"/>
    <arg name="tools_prompt" default="$(find tello_llm_ros)/config/prompts/pure_text_tools_description-EN.txt"/>

    <!-- llm node -->
    <node name="llm_service_node" pkg="tello_llm_ros" type="llm_service_node.py" output="screen">
        <param name="model_name" value="codellama:7b"/>
        <param name="model_type" value="ollama"/>
        <param name="timeout" value="100.0"/>
        <param name="common_system_prompt_file" value="$(arg common_prompt)"/>
        <param name="tools_description_file" value="$(arg tools_prompt)"/>
    </node>
</launch>